test/remote/gateways/remote_adyen_test.rb

```diff
@@ -1161,7 +1161,7 @@ def test_invalid_expiry_month_for_purchase
     card = credit_card('4242424242424242', month: 16)
     assert response = @gateway.purchase(@amount, card, @options)
     assert_failure response
-    assert_equal 'The provided Expiry Date is not valid.: Expiry month should be between 1 and 12 inclusive', response.message
+    assert_equal 'The provided Expiry Date is not valid.: Expiry month should be between 1 and 12 inclusive: 16', response.message
   end
```

Comment:
Unrelated to my change but it was an easy fix for a failing test.

Reply:
Thanks for updating!